aat/src/aat/resources/features/F-097 RoleToAccessProfiles/S-097.7.td.json

```diff
@@ -0,0 +1,30 @@
+{
+  "_guid_": "S-097.7",
+  "_extends_": "Import_Base_data",
+  "title": "Import Definition file without the CaseAccessCategories column in the RoleToAccessProfiles tab",
+  "specs": [
+    "an active profile in CCD",
+    "contains a missing CaseAccessCategories column"
+  ],
+  "request": {
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "file",
+          "filePath": "uk/gov/hmcts/ccd/test_definitions/invalid/BEFTA_Master_Definition_Missing_RoleToAccessProfiles_CaseAccessCategories_Column.xlsx"
```

Comment:
NB: We cannot have a successful import test of a different version of the *BEFTA_MASTER* file for this test as it would replace the common one. So we should delete this file from the corresponding hmcts/ccd-test-definitions release (we should have a PR for this) as the invalid folder should only contain invalid def files. And we can test using one of the other valid def files that is missing the column.

Reply:
Actioned in 8cb073dc0c2c6a82c68cf987eb7502b1309ad9ff

Reply:
👍 thanks for switch to valid def file path